byron/ledger/formal-spec/properties.tex

```diff
@@ -210,5 +210,100 @@ \subsection{UTxO Properties}
   we have:
 
   $$ \var{reserves} + \balance{utxo} =  \var{reserves_0} + \balance{utxo_0} $$
+\end{property}
+
+\subsection{Update Properties}
+\label{sec:update-properties}
+
+\subsubsection{PVBUMP Properties}
+\label{sec:pvbump-properties}
+
+Property~\ref{PVBUMP-empty-future-adopt} states that when no proposals are
+available, the system remains in the same state it started with.
 
+\begin{property}[PVBUMP no change on empty future adoptions]\label{PVBUMP-empty-future-adopt}
+  \textrm{For all environments such that $fads = \epsilon$:}
+
+  $$
+  \left(
+    \begin{array}{l}
+      \var{s_n}\\
+      \epsilon
+    \end{array}
+  \right)
+  \vdash
+  \left(
+    \begin{array}{l}
+      \var{pv}, \var{pps}\\
+    \end{array}
+  \right)
+  \trans{\hyperref[fig:rules:pvbump]{pvbump}}{}
+  \left(
+    \begin{array}{l}
+      \var{pv}, \var{pps}\\
+    \end{array}
+  \right)
+  $$
+\end{property}
+
+Property~\ref{prop:pvbump-early-on} states that early on in the lifetime
+of the blockchain the PVBUMP system remains in the same state it started with.
+
+\begin{property}[PVBUMP early on]\label{prop:pvbump-early-on}
```

Comment:
Same here: we need to quantify over `sn`, `fads`, `pv`, and `pps`.

Reply:
Fixed!